### ISSUE TITLE
dice loss

### ISSUE THREAD

**Issue by nnop** [nnop/notes#202]
- [1606.04797] V-Net: Fully Convolutional Neural Networks for Volumetric Medical Image Segmentation
- [1707.03237] Generalised Dice overlap as a deep learning loss function for highly unbalanced segmentations
